build.zig

```diff
@@ -36,7 +60,11 @@ pub fn build(b: *Builder) !void {
     });
     b.default_step.dependOn(&fmt_step.step);
 
-    const main_src = "src/kernel/kmain.zig";
+    const main_src = switch (target.getCpuArch()) {
+        .i386 => "src/kernel/kmain.zig",
+        .x86_64 => "src/kernel/kmain_64.zig",
```

Comment:
What I do in the aarch64 port is check for the architecture in kmain and skip certain parts of the initialisation if it's aarch64. You could consider doing that for x86_64 as well but either is fine as it's WIP.

Reply:
From offline discussion, until the 64 bit port is at the same level as the 32 bit port, these files will be merged as a lot of the kernel.zig code will need to be commented out for the 64 bit port to work. So creating a new kmain and using that will reduce the amount of commented code and make thing clearer.